python/mxnet/ndarray_doc.py

```diff
@@ -37,6 +37,17 @@ class ReshapeDoc(NDArrayDoc):
     (4L, 3L, 2L)
     """
 
+class elemwise_addDoc(NDArrayDoc):
```

Comment:
I'm not sure how to write this example in a language neutral way. Is there any other function that has a language neutral example that I can use as reference?